### PR TITLE
[homeconnect] Factorize handling of power command

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
@bruestel : I see the channel "power_state" for the cooktop (hob) with no handling of commands in the corresponding handler. Is it a read-only information for the cooktop or is something missing ?

File: bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/HomeConnectOvenHandler.java [hunk 1]
```diff
@@ -160,10 +160,7 @@ protected void handleCommand(final ChannelUID channelUID, final Command command,
         super.handleCommand(channelUID, command, apiClient);
 
         // turn coffee maker on and standby
```
Comment (bruestel) [inline bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/HomeConnectOvenHandler.java:162]: Could you please correct this comment. Copy and paste error by me.

File: bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/AbstractHomeConnectThingHandler.java [hunk 2]
```diff
@@ -1172,6 +1172,14 @@ protected void handleLightCommands(final ChannelUID channelUID, final Command co
         }
     }
 
+    protected void handlePowerCommand(final ChannelUID channelUID, final Command command,
+            final HomeConnectApiClient apiClient, String stateNotOn)
```
Comment (bruestel) [inline bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/AbstractHomeConnectThingHandler.java:1176]: Good idea. Should work for all appliances.
Bildschirmfoto-20210527175827-1142x240

**Review by bruestel — CHANGES_REQUESTED** [review]

**Comment by bruestel** [comment]
> > Would you be so kind and add it?
> 
> I must use OFF or standby ?

Please just remove `// turn coffee maker on and standby`

**Comment by lolodomo** [comment]
> 
> 
> > > Would you be so kind and add it?
> > 
> > 
> > I must use OFF or standby ?
> 
> Please just remove `// turn coffee maker on and standby`

I was asking for the cooktop. You told me you forgot th ehandling of the power command. So I will add it but I don't know if I should use OFF or Standby.

**Comment by lolodomo** [comment]
I choose ON+standby for the cooktop, let me know if this is correct.

**Comment by lolodomo** [comment]
Checking the API documentation, OFF must be considered for cooktop.

**Comment by bruestel** [comment]
> Checking the API documentation, OFF must be considered for cooktop.

yes, OFF is correct

**Comment by lolodomo** [comment]
@bruestel : please approve if this is now ok for you.

**Pull request merged**